src/components/TriggerEditor/index.tsx

```diff
@@ -446,7 +446,7 @@ export default ({
   return (
     <Row>
       <Col md={isSourceVisible ? 6 : 12}>
-        <Container fluid className="bg-white rounded p-3">
+        <Container as={Stack} gap={3} className="bg-white rounded p-3">
```

Comment:
using `as={Stack}` makes the container `fluid`?
Removing this class will give the container a max with on high resolution monitors, which is not well suited for this page